app/services/csv/lettings_log_csv_service.rb

```diff
@@ -329,5 +329,18 @@ def age_not_known?(log, attribute)
       age_known_field = PERSON_DETAILS.find { |key, _value| key == attribute }[1]["age_known_field"]
       log[age_known_field] == 1
     end
+
+    def non_question_fields
+      case @year
+      when 2022
+        %w[id status assigned_to is_dpo created_at updated_by updated_at creation_method old_id old_form_id collection_start_year created_by]
+      when 2023
+        %w[id status duplicate_set_id assigned_to is_dpo created_at updated_by updated_at creation_method old_id old_form_id collection_start_year created_by]
+      when 2024
+        %w[id status duplicate_set_id assigned_to is_dpo created_at updated_by updated_at creation_method collection_start_year bulk_upload_id address_line1_as_entered address_line2_as_entered town_or_city_as_entered county_as_entered postcode_full_as_entered la_as_entered created_by]
+      else
+        %w[id status duplicate_set_id assigned_to is_dpo created_at updated_by updated_at creation_method collection_start_year bulk_upload_id address_line1_as_entered address_line2_as_entered town_or_city_as_entered county_as_entered postcode_full_as_entered la_as_entered created_by]
+      end
```

Comment:
Is there a nice way to avoid this being a long case statement that we (presumably) need to add to every year? (I suspect there may not be)

Reply:
I thought about piecing it together, but it becomes less readable. It shouldn't get larger than this provided we don't decide to increase the number of forms we accommodate at any given time